_docs/getting-started/010-drill-introduction.md

```diff
@@ -10,6 +10,13 @@ applications, while still providing the familiarity and ecosystem of ANSI SQL,
 the industry-standard query language. Drill provides plug-and-play integration
 with existing Apache Hive and Apache HBase deployments.   
 
+## What's New in Apache Drill 1.19
```

Comment:
A lot of features are missing and I will submit comments later.

Reply:
Does all version changes can be actually considered as new features?

Reply:
Added all the new features mentioned below, left aside the version changes

Reply:
Everything is okay. Just, The Mongo (4.4) and Kafka (2.8) storage plugin have had a major upragde than before.